subprojects/studio-plugin/src/test/groovy/org/gradle/profiler/studio/plugin/StudioPluginTestCase.groovy

```diff
@@ -0,0 +1,76 @@
+package org.gradle.profiler.studio.plugin
+
+import com.intellij.openapi.application.WriteAction
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.project.ex.ProjectManagerEx
+import com.intellij.openapi.projectRoots.ProjectJdkTable
+import com.intellij.openapi.projectRoots.Sdk
+import com.intellij.testFramework.EdtTestUtil
+import com.intellij.testFramework.HeavyPlatformTestCase
+import com.intellij.testFramework.OpenProjectTaskBuilder
+import org.gradle.profiler.client.protocol.Server
+import org.jetbrains.annotations.NotNull
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+import java.nio.file.Path
+
+@RunWith(JUnit4.class)
+class StudioPluginTestCase extends HeavyPlatformTestCase {
```

Comment:
This could be moved to test-fixtures, but I would have to figure out exactly what test dependencies does intellij-gradle-plugin applies. So for now I left it in tests folder.

Reply:
The plan is to resolve this before merging the PR, right?

Reply:
Will check it

Reply:
Done. Moved it to test fixtures.